docs/docs/3-features-modules/9-brevo-module/1-setup.md

```diff
@@ -0,0 +1,692 @@
+---
+title: Brevo Setup
+---
+
+# Setup
+
+:::caution
+This documentation refers to Brevo v3 or higher.
+Make sure that your project uses COMET v7.0 or later.
+:::
+
+The Brevo Module provides three packages: `brevo-api`, `brevo-admin`, and the optional `brevo-mail-rendering` package. Please check the latest release [here](https://github.com/vivid-planet/comet-brevo-module/releases).
+
+## API
+
+### Installation
+
+To add the Brevo API package, add the following to your `package.json` dependencies and install:
+
+```json
+"@vivid-planet/comet-brevo-api": "^3.0.0"
+```
+
+### Config
+
+You need to provide configuration values so your application can connect to Brevo and related services.  
+Replace the `envVars.*` placeholders with your actual environment variable names or values.
+
+Example configuration:
+
+```
+brevo: {
+    apiKey: envVars.BREVO_API_KEY,
+    redirectUrlForImport: envVars.REDIRECT_URL_FOR_IMPORT,
+},
+campaign: {
+    url: envVars.CAMPAIGN_URL,
+    basicAuth: {
+        username: envVars.CAMPAIGN_BASIC_AUTH_USERNAME,
+        password: envVars.CAMPAIGN_BASIC_AUTH_PASSWORD,
+    },
+},
+ecgRtrList: {
+    apiKey: envVars.ECG_RTR_LIST_API_KEY,
+},
+```
+
+See the table below for information on how to find the config values and what they are used for:
+
+| Config Key                    | Description                                                                                                                      |
+| ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `brevo.apiKey`                | Your Brevo API key. Find it in your [Brevo account settings](https://app.brevo.com/settings/keys/api).                           |
+| `brevo.redirectUrlForImport`  | The URL in your app to redirect to after importing contacts. Set this to a route in your application.                            |
+| `campaign.url`                | The URL where the campaign is rendered, used both for admin preview and for generating the final HTML before it’s sent to Brevo. |
+| `campaign.basicAuth.username` | Username for HTTP Basic Auth if your campaign frontend is protected. Set this yourself if needed.                                |
+| `campaign.basicAuth.password` | Password for HTTP Basic Auth if your campaign frontend is protected. Set this yourself if needed.                                |
+| `ecgRtrList.apiKey`           | API key for the ECG RTR List integration. Obtain this from the respective service or your administrator.                         |
+
+### Register module
+
+To use the Brevo Module in your project, you need to register it in your main `AppModule`.  
+Pass the `scope` to the `resolveConfig` function to use the appropriate values for each scope.  
```

Comment:
Should we mention that this is only necessary if the config differs based on the scope.